test/test_transcript_support_level.py

```diff
@@ -0,0 +1,22 @@
+"""
+Tests for methods which return collections of transcript IDs that aren't
+converting from some type of name or ID.
+"""
+from __future__ import absolute_import
+
+from nose.tools import eq_
+
+from pyensembl import cached_release
+
+def test_transcript_support_level():
+    ensembl93 = cached_release(93)
+    transcript = ensembl93.transcripts_by_name("DDX11L1-202")[0]
+    eq_(transcript.support_level, "1")
+    # for this transcript, the tsl value is missing, and should yield an empty string:
+    transcript = ensembl93.transcripts_by_name("OR4G11P-202")[0]
```

Comment:
Do you have any idea why/when the support level is missing for transcripts? Also is it desirable to empty strings as a distinct missing value from `None`?

Reply:
I think it might be missing for pseudogenes (and in this case, processed transcripts):

> Transcript Support Level Not Analysed. Pseudogenes, single exon transcripts, HLA, T-cell receptor and Ig transcripts are not analysed and therefore not given any of the TSL categories.

An empty string is what the db returns on a missing value of an existing column. My reasoning was that an empty string allows to distinguish a missing value from a missing column. On the other hand, a use can always check with `db.column_exists()`, so I think I'll just change it to None.